include/gazebo_mavlink_interface.h

```diff
@@ -66,13 +66,13 @@
 
 #include <ignition/transport/Node.hh>
 #include <ignition/msgs/imu.pb.h>
-#include <msgs/Pressure.pb.h>
-#include <msgs/MagneticField.pb.h>
-#include <msgs/SITLGps.pb.h>
+#include <Pressure.pb.h>
+#include <MagneticField.pb.h>
+#include <SITLGps.pb.h>
 
 #include <common.h>
 
-#include <development/mavlink.h>
+#include <mavlink/v2.0/development/mavlink.h>
```

Comment:
I think this will break the path if it is being built as part of the PX4 firmware. Was the intention trying to follow the convention of colcon packages?

Reply:
OK, was hoping it would still work on that side of things. I just changed this to match how the cmake find module for mavlink was working. I can probably add an extra include_dir in the find script so this just works.